ui/src/modules/panel/components/stats.js

```diff
@@ -11,6 +11,17 @@
 
 import { html, msg, router, dispatch } from 'hybrids';
 
+function openTabWithUrl(host, event) {
```

Comment:
isn't this redundant?

Reply:
Yes, this is intentional. This code is repeated in separate packages. It would be placed somewhere in `ui` utils and exported if shared. I thought that it is not worth it.